artman/cli/main.py

```diff
@@ -417,3 +418,23 @@ def _print_log(pipeline_id):
         '    gcloud beta logging read "logName=projects/vkit-pipeline/logs/%s"'
         % pipeline_id,
     )
+
+
+def _chown_for_artman_output():
```

Comment:
Can `g3artman` just set this automatically? It knows who the user and group are.